examples/rules/data/stories.md

```diff
@@ -1,73 +1,58 @@
 <!-- each story will be perceived as independent rule -->
 
-
-## activate loop q_form
+>> activate loop q_form
 <!-- required slots for q_form should be listed somewhere else -->
     - ...
 * activate_q_form  <!-- like request_restaurant -->
-    - action_activate_q_form  <!-- default action with an utterance like action_restart -->
-    - form{"name": "q_form"} <!-- problem with predicting this event, because, it has to be used as condition -->
-    - action_loop_q_form  <!-- can be anything -->
+    - loop_q_form  <!-- Activate form -->
```

Comment:
need to add form event afterwards, to indicate that `action_listen` should be predicted only if form is still active
```suggestion
    - loop_q_form  <!-- Activate form -->
    - form{"name": "loop_q_form"}
```